content/4_Gold/Combinatorics.mdx

```diff
@@ -887,6 +887,49 @@ int main() {
 	}
 }
 ```
+## Expected Value
+
+An expected value is the theoretical mean of a probability distribution. A random variable is used to represent a possible probability distribution. Let $X$ be a random variable and $P(X = x)$ be the probability that the result of the random variable $X$ is $x$. Then, the expected value of $X$, denoted as $E[X]$, is
```

Comment:
italize "random variable" so they know its a term

Reply:
i assume you mean bold cause that is what i see everywhere else but ok